src/test/java/org/candlepin/service/impl/DefaultEntitlementCertServiceAdapterTest.java

```diff
@@ -995,8 +1002,8 @@ public void testPrepareV1ExtensionsBrandedProduct() {
         products.add(product);
         setupEntitlements(ARCH_LABEL, "1.0");
 
-        Set<X509ExtensionWrapper> extensions =
-            certServiceAdapter.prepareV1Extensions(products, pool, consumer,
+        Set<X509ExtensionWrapper> extensions = certServiceAdapter.prepareV1Extensions(products, pool,
```

Comment:
Another example of a busted spotless rule

Reply:
To prevent this one, we can enable joining lines, but we would loose all control over line splitting. I disabled the rule to give us option to have custom line breaks.